demo/plugins.txt

```diff
@@ -9,7 +9,10 @@ git-server:1.6
 handlebars:1.1.1
 icon-shim:2.0.3
 jquery-detached:1.2.1
+lockable-resources:1.8
+mailer:1.5
```

Comment:
Transitive dependencies of `lockable-resources`. All dependencies must be explicitly set as there is no dependencies resolution here, no?